webhook-bridge/svix-webhook-bridge/Cargo.toml

```diff
@@ -8,9 +8,41 @@ edition = "2021"
 [dependencies]
 anyhow = "1"
 clap = { version = "4.2.4", features = ["env", "derive"] }
-# FIXME: latest deno has been seen to leak. Watch the memory profile.
-deno_core = "0.185.0"
-deno_runtime = "0.111"
+
+# Latest deno: leaks.
+#deno_core = "0.185.0"
+#deno_runtime = "0.111.0"
+
+# This is just after what og-ingester was using (tagged 1.29.4).
+# The actuals: 0.165 and 0.91 (tagged 1.29.2). These fail to resolve deps - conflicts on tokio and related.
+# Still leaks!
+#deno_core = "0.167"
+#deno_runtime = "0.93"
+
+# These are "old" deno - about a year old, v1.22.0.
+# Worker API is different here. Not clear on how to get values back from a script execution in a worker; returns only unit.
+#deno_core = "0.135.0"
+#deno_runtime = "0.61.0"
+
+# This is v1.30.3 - the bug reports of leaks started in 1.31.x
+#deno_core = "0.171.0"
+#deno_runtime = "0.97.0"
+
+# tagged 1.27.2 (leaks)
+#deno_core = "0.158.0"
+
+# tagged 1.26.2 (leaks)
+#deno_core = "0.155.0"
+
+# tagged 1.25.4
+#deno_core = "0.152.0"
+
+# tagged 1.24.3 (leaks, but not quite as badly as later releases - backtrace showed v8 GC frames)
+#deno_core = "0.147.0"
```

Comment:
I can cut all this out. I was keeping notes as I went, profiling the memory for each minor at its highest patch-level.